xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/v3/AccountTransactionsIT.java

```diff
@@ -58,19 +63,27 @@ public void listTransactionsPagination() throws JsonRpcClientErrorException {
     assertThat(results.marker()).isNotEmpty();
 
     int transactionsFound = results.transactions().size();
-    int pages = 1;
-    while (results.marker().isPresent()) {
+    int pages = 0;
+    while (results.marker().isPresent()
+      // Needed because clio is broken. See https://github.com/XRPLF/clio/issues/195#issuecomment-1247412892
+      && results.transactions().size() > 0
+    ) {
       results = mainnetClient.accountTransactions(AccountTransactionsRequestParams.builder(minLedger, maxLedger)
         .account(MAINNET_ADDRESS)
+        .limit(UnsignedInteger.valueOf(200L))
         .marker(results.marker().get())
         .build());
       assertThat(results.transactions()).isNotEmpty();
       transactionsFound += results.transactions().size();
       pages++;
+      logger.info("Retrieved {} ledgers (marker={} page={})",
+        transactionsFound,
+        results.marker().map($ -> $.value()).orElseGet(() -> "n/a"),
+        pages
+      );
     }
```

Comment:
```suggestion
    while (results.marker().isPresent()
      // Needed because clio is broken. See https://github.com/XRPLF/clio/issues/195#issuecomment-1247412892
      && results.transactions().size() > 0
    ) {
      results = mainnetClient.accountTransactions(AccountTransactionsRequestParams.builder(minLedger, maxLedger)
        .account(MAINNET_ADDRESS)
        .limit(UnsignedInteger.valueOf(200L))
        .marker(results.marker().get())
        .build());
      assertThat(results.transactions()).isNotEmpty();
      transactionsFound += results.transactions().size();
      pages++;
      logger.info("Retrieved {} ledgers (marker={} page={})",
        transactionsFound,
        results.marker().map($ -> $.value()).orElseGet(() -> "n/a"),
        pages
      );
    }
```
Should look like this instead: 
```
    while (results.marker().isPresent()) {
      results = mainnetClient.accountTransactions(AccountTransactionsRequestParams.builder(minLedger, maxLedger)
        .account(MAINNET_ADDRESS)
        .limit(UnsignedInteger.valueOf(200L))
        .marker(results.marker().get())
        .build());
      // Needed because clio is broken. See https://github.com/XRPLF/clio/issues/195#issuecomment-1247412892
      if (results.transactions().isEmpty()) {
        break;
      }
      assertThat(results.transactions()).isNotEmpty();
      transactionsFound += results.transactions().size();
      pages++;
      logger.info("Retrieved {} ledgers (marker={} page={})",
        transactionsFound,
        results.marker().map($ -> $.value()).orElseGet(() -> "n/a"),
        pages
      );
    }
```

Reply:
the first fetch would never return empty list of transactions. The last one would and hence should be checked inside in the while loop

Reply:
Yeah Sure!

Reply:
Updated: ```
    while (results.marker().isPresent()) {
      results = mainnetClient.accountTransactions(AccountTransactionsRequestParams.builder(minLedger, maxLedger)
        .account(MAINNET_ADDRESS)
        .limit(UnsignedInteger.valueOf(200L))
        .marker(results.marker().get())
        .build());
      // Needed because clio is broken. See https://github.com/XRPLF/clio/issues/195#issuecomment-1247412892
      if (results.transactions().isEmpty()) {
        break;
      }
      transactionsFound += results.transactions().size();
      pages++;
      logger.info("Retrieved {} transactions (marker={} page={})",
        transactionsFound,
        results.marker().map($ -> $.value()).orElseGet(() -> "n/a"),
        pages
      );
    }
```

Reply:
Why do you need 
```
if (results.transactions().isEmpty()) { 
  break;
}
```

if you don't care if it's empty or not?

Reply:
Again!
```
    while (results.marker().isPresent()
      // Needed because clio is broken. See https://github.com/XRPLF/clio/issues/195#issuecomment-1247412892
      && results.transactions().size() > 0
    ) {
      results = mainnetClient.accountTransactions(AccountTransactionsRequestParams.builder(minLedger, maxLedger)
        .account(MAINNET_ADDRESS)
        .limit(UnsignedInteger.valueOf(200L))
        .marker(results.marker().get())
        .build());
      transactionsFound += results.transactions().size();
      pages++;
      logger.info("Retrieved {} ledgers (marker={} page={})",
        transactionsFound,
        results.marker().map($ -> $.value()).orElseGet(() -> "n/a"),
        pages
      );
    }
```

Reply:
Thanks @nkramer44 !

Reply:
I just heard that it should be deployed before EOD and shoul dbe deployed to Clio. Let's wait till then and decide tomorrow!

Reply:
+1 to this change.